src/main/java/bh/bot/common/types/AttendablePlaces.java

```diff
@@ -14,6 +14,7 @@ public static class Ids {
         public static final int Pvp         = 0b00100000;
         public static final int WorldBoss   = 0b01000000;
         public static final int Raid        = 0b10000000;
+        public static final int Quest       = 0b00000011;
```

Comment:
This conflict with Invasion (0b1) + Trials (0b10) and will cause problem with select mode manually via command prompt. Please migrate to use long so we will have 8 bytes instead of just 4 byte of int